utils/utils.c

```diff
@@ -280,14 +281,111 @@ void fetchGlfsVolServerFromEnv()
   LOG("mgmt", GB_LOG_INFO, "Block Hosting Volfile Server Set to: %s", gbConf.volServer);
 }
 
+
+static int
+glusterLogrotateConfigSet(char *logDir)
+{
+  char *buf = NULL, *line = NULL, *p, *dom = NULL;
+  int ret, m, len;
+  size_t n;
+  FILE *fp;
+
+
+  if (gbCtx == GB_CLI_MODE) {
+      dom = "cli";
+  } else if (gbCtx == GB_DAEMON_MODE) {
+      dom = "mgmt";
+  }
+
+  if (!dom) {
+    return -EINVAL;
+  }
+
+  fp = fopen(GB_LOGROTATE_PATH, "r+");
+  if (fp == NULL) {
+    ret = -errno;
+    LOG(dom, GB_LOG_ERROR, "Failed to open file '%s', %m", GB_LOGROTATE_PATH);
```

Comment:
@lxbsz One more thing:

instead of %m, use stderror(errno)

I remember there was a patch to remove all such uses, may be a co-verity fix.

Reply:
Will fix it.
Thanks.